addons/hr_timesheet/models/project.py

```diff
@@ -8,7 +8,14 @@
 class Project(models.Model):
     _inherit = "project.project"
 
-    allow_timesheets = fields.Boolean("Timesheets", default=True, help="Enable timesheeting on the project.")
+    allow_timesheets = fields.Boolean(
+        "Timesheets",
+        compute='_compute_allow_timesheets',
+        store=True,
+        readonly=False,
+        default=True,
+        help="Enable timesheeting on the project.")
```

Comment:
I know there aren't any guidelines about line splitting. But It is a bit too much IMHO

Reply:
My point of view: putting 6 parameters on a same line or on 2 lines is not readable. When I cannot put all the parameters on a same line without ensuring a good readability, I prefer writing one parameter by line with the parameter name, so it's really easy to see without thinking which parameters are set and which are the values.

For me, almost all the field definitions in Odoo models are unreadable ... And I always keep in my that a code is written once but read a lot of times :-) But I'm open to a solution that takes care of readability more that compacity :-)